DelaunayTriangulationSwift/Vertex.swift

```diff
@@ -6,46 +6,45 @@
 //  Copyright © 2016 zero. All rights reserved.
 //
 
+import CoreGraphics
+
 public struct Vertex {
     
     public init(x: Double, y: Double) {
         self.x = x
         self.y = y
     }
     
+    public func pointValue() -> CGPoint {
+        return CGPoint(x: x, y: y)
+    }
+    
     public let x: Double
     public let y: Double
 }
 
-extension Vertex: Equatable {
-    static public func ==(lhs: Vertex, rhs: Vertex) -> Bool {
-        return lhs.x == rhs.x && lhs.y == rhs.y
-    }
-}
+extension Vertex: Equatable { }
 
-extension Vertex: Hashable {
-    public var hashValue: Int {
-        return "\(x)\(y)".hashValue
-    }
+public func ==(lhs: Vertex, rhs: Vertex) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
 }
```

Comment:
This method of implementing the equatable extension feels dirty to me. The implementation is separate from the conformance to the protocol that requires it whereas in the previous code it was together.